[2025-05-28 23:58:03,696] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: TwitterDAG.twitter_datascience manual__2025-05-29T02:58:00.317851+00:00 [queued]>
[2025-05-28 23:58:03,712] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: TwitterDAG.twitter_datascience manual__2025-05-29T02:58:00.317851+00:00 [queued]>
[2025-05-28 23:58:03,712] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2025-05-28 23:58:03,712] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2025-05-28 23:58:03,712] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2025-05-28 23:58:03,744] {taskinstance.py:1377} INFO - Executing <Task(TwitterOperator): twitter_datascience> on 2025-05-29 02:58:00.317851+00:00
[2025-05-28 23:58:03,750] {standard_task_runner.py:52} INFO - Started process 18766 to run task
[2025-05-28 23:58:03,770] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'TwitterDAG', 'twitter_datascience', 'manual__2025-05-29T02:58:00.317851+00:00', '--job-id', '13', '--raw', '--subdir', 'DAGS_FOLDER/twitter_dag.py', '--cfg-path', '/tmp/tmpbdrojlfp', '--error-file', '/tmp/tmp2vktw79e']
[2025-05-28 23:58:03,778] {standard_task_runner.py:80} INFO - Job 13: Subtask twitter_datascience
[2025-05-28 23:58:03,990] {task_command.py:370} INFO - Running <TaskInstance: TwitterDAG.twitter_datascience manual__2025-05-29T02:58:00.317851+00:00 [running]> on host alura
[2025-05-28 23:58:04,127] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=TwitterDAG
AIRFLOW_CTX_TASK_ID=twitter_datascience
AIRFLOW_CTX_EXECUTION_DATE=2025-05-29T02:58:00.317851+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2025-05-29T02:58:00.317851+00:00
[2025-05-28 23:58:04,138] {base.py:68} INFO - Using connection ID 'twitter_default' for task execution.
[2025-05-28 23:58:04,141] {twitter_hook.py:34} INFO - URL: https://labdados.com/2/tweets/search/recent?query=datascience&tweet.fields=author_id,conversation_id,created_at,id,in_reply_to_user_id,public_metrics,lang,text&expansions=author_id&user.fields=id,name,username,created_at&start_time=2025-05-28T00:00:00.00Z&end_time=2025-05-29T00:00:00.00Z
[2025-05-28 23:58:05,003] {taskinstance.py:1395} INFO - Marking task as SUCCESS. dag_id=TwitterDAG, task_id=twitter_datascience, execution_date=20250529T025800, start_date=20250529T025803, end_date=20250529T025805
[2025-05-28 23:58:05,093] {local_task_job.py:156} INFO - Task exited with return code 0
[2025-05-28 23:58:05,145] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
